### PR TITLE
Removed support for selfie video widget

### LINKED ISSUE #5467: Get rid of the selfie mode in Video widgets
#### Problem description
Selfie mode in Video widgets seems to be an unused feature that only takes our time to maintain and fix bugs/crashes. According to our analytics, only **20 users** used it in the last two months
image
**17** of them used it in the AllWidgets form. That means there were only **3 users** that used that appearance in their own form and they generated only **4 events** what indicates they maybe just tested that option or used it by accident.

It looks like we can safely remove that functionality.

### PR DESCRIPTION
**Pull request by grzesiek2010**
Closes #5467 

#### What has been done to verify that this works as intended?
I've verified that the `selfie` appearance is now ignored in the Video widget and it works like without any appearance.

#### Why is this the best possible solution? Were any other approaches considered?
As discussed in the issue. This was an unused feature that only required our work to maintain it.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
Please verify that Video and Image widgets work well with and without the `selfie` appearance. Other parts of the app should not be affected.

#### Do we need any specific form for testing your changes? If so, please attach one.
AllWidgets form.

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
Yes https://github.com/getodk/docs/issues/1583
I've also update the AllWidgets form https://docs.google.com/spreadsheets/d/1af_Sl8A_L8_EULbhRLHVl8OclCfco09Hq2tqb9CslwQ/edit#gid=0

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: selfie-camera/src/main/java/org/odk/collect/selfiecamera/Camera.kt [hunk 1]
```diff
@@ -19,9 +19,3 @@ internal interface Camera {
 internal interface StillCamera : Camera {
```
Comment (seadowg) [inline selfie-camera/src/main/java/org/odk/collect/selfiecamera/Camera.kt]: I'm thinking we should just push up the `StillCamera` interface into `Camera`. There's no need for the two layers in the hierarchy any longer.

**Review by seadowg — CHANGES_REQUESTED** [review]
Happy to see this 😆. I've got one change I'd like to see, but it shouldn't block QA.

File: selfie-camera/src/main/java/org/odk/collect/selfiecamera/Camera.kt (hunk 1 above)
Comment (grzesiek2010) [reply selfie-camera/src/main/java/org/odk/collect/selfiecamera/Camera.kt]: Right, done.

**Comment by dbemke** [comment]
**Tested with Success!**

Verified on Android 8.1, 10, 13 emulated Android 6.0

**Verified Cases:**
- selfie video widget works like video widget
- removing, editing, submitting pictures and vidoes
- high res video enabled/disabled
- don’t keep acitivities enabled/disabled
- rotating, minimizing, locking the screen
- regression checks in Image and Media widgets

**Review by seadowg — APPROVED** [review]

**Pull request merged**